src/components/MessageManager/MessageManager.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import "./MessageManager.css";
+import MessagingWindow from "../MessagingWindow/MessagingWindow";
+
+import Firebase from "firebase";
+
+class MessageManager extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      friends: [],
+      friendClicked: {},
+      user: this.props.user
```

Comment:
it seems like storing user in state here is useless because its coming from props